src/resources/generated/apps_secret.rs

```diff
@@ -22,7 +22,6 @@ pub struct AppsSecret {
     pub deleted: Option<bool>,
 
     /// The Unix timestamp for the expiry time of the secret, after which the secret deletes.
-    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Just using this line as a general example, do you know why all these `skip_serializing_if` are now gone? Unless the previous behavior was wrong in some way, that change might result in sending unnecessarily larger request payloads